### PR TITLE
BFF: Implement delegation endpoint

### PR DESCRIPTION
**Pull request by allinox**
## Description

This PR implements the delegation endpoint in the BFF with a functioning mock and tests.

In addition to this, it adds a few small changes:
- Updates ServiceResource model
- Filters service-getters so that only resources with the flags visible and delegable are given to frontend
- Changes the DelegationOutput model so that it actions are only given by a single string such as in checkAccess-endpoint
- Updates dependecies Designsystem and designsystem tokens
- A small visible fix on light colored Actionbar to display border on small size

## Related Issue(s)
- #396 

## Verification
- [ ] **Your** code builds clean without any errors or warnings
- [ ] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [ ] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Mocks/Mocks/SingleRightClientMock.cs [hunk 1]
```diff
@@ -2,30 +2,69 @@
...
+                    break;
+                default:
+                    resourceFileName = "Unknown";
+                    break;
+            }
+
+            string dataPath = Path.Combine(localPath, "Data", "SingleRight", "CreateDelegation");
+            try
+            {
+                string data = Util.GetMockDataSerialized(dataPath, resourceFileName + ".json");
```
Comment (Albertlarsen) [inline backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Mocks/Mocks/SingleRightClientMock.cs:57]: Does it not work to use GetMockData and specify it should be for a string?

File: backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Mocks/Altinn.AccessManagement.UI.Mocks.csproj [hunk 2]
```diff
@@ -12,6 +12,15 @@
     </Content>
   </ItemGroup>
 
+  <ItemGroup>
+    <None Remove="Data\SingleRight\CreateDelegation\3225.json" />
```
Comment (Albertlarsen) [inline backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Mocks/Altinn.AccessManagement.UI.Mocks.csproj:16]: What are these for?

Comment (allinox) [reply backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Mocks/Altinn.AccessManagement.UI.Mocks.csproj:16]: Hmm, I think they were added automatically by visual studio, but I don't think we really need to specify them all independently like this so I'll just remove it

File: backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Tests/Controllers/SingleRightCotrollerTest.cs [hunk 3]
```diff
@@ -164,9 +169,285 @@ public async Task DelegationAccessCheck_ReadAndWrite_valid_response()
...
+            // Arrange
+            string partyId = "999 999 999";
+            string toSsn = "50019992";
+
+            List<IdValuePair> resource = new List<IdValuePair>
+            {
+                new IdValuePair
+                {
+                    Id = "urn:altinn:resource",
+                    Value = "NonExistant",
```
Comment (Albertlarsen) [inline backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Tests/Controllers/SingleRightCotrollerTest.cs:400]: Nonexistent*

**Review by Albertlarsen — APPROVED** [review]
Nice stuff 😊

**Pull request merged**